content/7-webapp-features/1-fetch-data/ember/users.hbs

```diff
@@ -0,0 +1,10 @@
+<ul>
+    {{#each this.allUsers as |user|}}
+        <li key={user.login.uuid}>						
```

Comment:
```suggestion
        <li>						
```

Seems wrongly copy pasted from the React example.
If you want to use a key, it should be added to the `{{#each}}` block:


```hbs
{{#each this.allUsers key="login.uuid" as |user|}}
```

Reply:
Yeah, my bad. I will make the required changes